stl.go

```diff
@@ -707,6 +694,24 @@ func newTTIBlock(i *Item, idx int) (t *ttiBlock) {
 	return
 }
 
+func stlJustificationCodeFromStyle(sa *StyleAttributes) byte {
+	if sa == nil || sa.STLPosition == nil {
```

Comment:
I think you want to check `sa.STLJustification == nil` instead of `sa.STLPosition == nil`

Reply:
Duh sorry 🤦